plugins/PagePerformance/Columns/TimeDomCompletion.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\PagePerformance\Columns;
+
+use Piwik\Columns\DimensionMetricFactory;
+use Piwik\Columns\MetricsList;
+use Piwik\Piwik;
+use Piwik\Plugin\ArchivedMetric;
+use Piwik\Plugin\ComputedMetric;
+use Piwik\Plugin\Dimension\ActionDimension;
+use Piwik\Tracker\Action;
+use Piwik\Tracker\ActionPageview;
+use Piwik\Tracker\Request;
+use Piwik\Tracker\Visitor;
+
+class TimeDomCompletion extends ActionDimension
+{
+    protected $columnName = 'time_dom_completion';
+    protected $columnType = 'INTEGER(10) UNSIGNED NULL';
```

Comment:
Are we storing values as milliseconds? Then maybe a mediumint would still do? Considering `16777215 ms = 280 minutes`... also for the other dimensions

Reply:
Didn't spent too much time on thinking about the field type. Will change it to mediumint